python/astro_metadata_translator/translator.py

```diff
@@ -581,9 +582,15 @@ def determine_translator(
                 log.debug("Using translation class %s%s", name, file_msg)
                 return trans
 
+        plugins = [p.name for p in entry_points(group="astro_metadata_translators")]
```

Comment:
In theory we could make the entry point function return a list of all the translator names that are supported by the plugin. This would allow us to form a more accurate error message where we wouldn't suggest obs_lsst if the obs_lsst translators were already listed, and we could have the message include the mapping from label to translator name (which we could also do with the `--list-plugins` option). This would slow down error reporting but be much more informative. cc/ @gpdf

Reply:
Given it can take 5 seconds to load the plugins I made it so that `--list-plugins` has that overhead (they've requested it) but error reporting doesn't.